samples/AspNetCore/ControllerSample/Controllers/SampleController.cs

```diff
@@ -80,5 +80,20 @@ public async Task<ActionResult<Account>> Withdraw(Transaction transaction, [From
             await state.SaveAsync();
             return state.Value;
         }
+
+        /// <summary>
+        /// Method for throwing a BadRequest exception which will cause Dapr sidecar to throw an RpcException
+        /// </summary>
+        /// <param name="transaction">Transaction info.</param>
+        /// <param name="daprClient">State client to interact with Dapr runtime.</param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        [HttpPost("throwException")]
+        public async Task<ActionResult<Account>> ThrowException(Transaction transaction, [FromServices] DaprClient daprClient)
+        {
+            Console.WriteLine("Enter ThrowException");
+            var task = Task.Delay(10);
+            await task;
+            return BadRequest("{statusCode: 404, message: not found...}");
```

Comment:
yeah this is a little confused. If you want to return a 404, then I'd suggest `NotFound()`.

If you want to specify some content use `NotFound(new { message: "not found" });` and it will get JSON serialized. 

ASP.NET Core will return a default JSON blob for errors when you have `[ApiController]` on the class and don't specify your own response body - using problem details.